app/services/workflow_executions/create_service.rb

```diff
@@ -60,5 +61,21 @@ def sanitize_workflow_param(property, value)
         value
       end
     end
+
+    def create_activities
+      return unless @workflow_execution.submitter.automation_bot?
+
+      @workflow_execution.namespace.create_activity key: 'workflow_execution.automated_workflow.launch',
+                                                    owner: current_user,
+                                                    parameters:
+                                                    {
+                                                      workflow_id: @workflow_execution.id,
+                                                      workflow_name: @workflow_execution.name,
+                                                      sample_id:
+                                                      @workflow_execution.samples_workflow_executions.first.sample.id,
+                                                      sample_name:
```

Comment:
**nitpick (blocking):** Please rename this to `sample_puid`

Reply:
Updated in 032cb1c08ee993dee5d18b74ef198b03bafe07b3